packages/grid/_modules_/grid/hooks/features/keyboard/useGridKeyboard.ts

```diff
@@ -150,7 +155,10 @@ export const useGridKeyboard = (
 
   const handleCellKeyDown = React.useCallback(
     (params: GridCellParams, event: React.KeyboardEvent) => {
```

Comment:
handleCellKeyDown is just for cell, you should do this in `handleKeyDown`

Reply:
updated